### ISSUE TITLE
cannot setup master branch(with kotlinVersion 1.3.30-dev-532)

### ISSUE THREAD

**Issue by furyu-john** [JetBrains/kotlin-native#2513]
I run below task but get one error. 
```
$ ./gradlew dependencies:update
w: /Users/satohjohn/sandbox/kotlin/git-kotlin-native/kotlin-native/buildSrc/plugins/src/main/kotlin/org/jetbrains/kotlin/ExecutorService.kt: (125, 52): Unnecessary non-null assertion (!!) on a non-null receiver of type ExecResult
w: /Users/satohjohn/sandbox/kotlin/git-kotlin-native/kotlin-native/shared/src/library/kotlin/org/jetbrains/kotlin/konan/library/impl/KonanLibraryImpl.kt: (53, 34): The corresponding parameter in the supertype 'KonanLibrary' is named 'fqName'. This may cause problems when calling this function with named arguments.
w: /Users/satohjohn/sandbox/kotlin/git-kotlin-native/kotlin-native/shared/src/main/kotlin/org/jetbrains/kotlin/konan/util/WithLogger.kt: (9, 17): Parameter 'message' is never used

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/satohjohn/sandbox/kotlin/git-kotlin-native/kotlin-native/backend.native/build.gradle' line: 152

* What went wrong:
A problem occurred evaluating project ':backend.native'.
> Could not resolve all files for configuration ':backend.native:kotlin_common_stdlib_src'.
   > Could not find org.jetbrains.kotlin:kotlin-stdlib-common:1.3.30-dev-532.
     Searched in the following locations:
       - https://cache-redirector.jetbrains.com/maven-central/org/jetbrains/kotlin/kotlin-stdlib-common/1.3.30-dev-532/kotlin-stdlib-common-1.3.30-dev-532.pom
       - https://cache-redirector.jetbrains.com/maven-central/org/jetbrains/kotlin/kotlin-stdlib-common/1.3.30-dev-532/kotlin-stdlib-common-1.3.30-dev-532-sources.jar
       - https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_CompilerAllPlugins),number:1.3.30-dev-532,pinned:false/artifacts/content/maven/org/jetbrains/kotlin/kotlin-stdlib-common/1.3.30-dev-532/kotlin-stdlib-common-1.3.30-dev-532.pom
       - https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_CompilerAllPlugins),number:1.3.30-dev-532,pinned:false/artifacts/content/maven/org/jetbrains/kotlin/kotlin-stdlib-common/1.3.30-dev-532/kotlin-stdlib-common-1.3.30-dev-532-sources.jar
       - https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_130_CompilerAllPlugins),number:1.3.0-rc-116,tag:kotlin-native,pinned:true/artifacts/content/maven/org/jetbrains/kotlin/kotlin-stdlib-common/1.3.30-dev-532/kotlin-stdlib-common-1.3.30-dev-532.pom
       - https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_130_CompilerAllPlugins),number:1.3.0-rc-116,tag:kotlin-native,pinned:true/artifacts/content/maven/org/jetbrains/kotlin/kotlin-stdlib-common/1.3.30-dev-532/kotlin-stdlib-common-1.3.30-dev-532-sources.jar
     Required by:
         project :backend.native

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.10.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 27s
```

So I was looked 'https://teamcity.jetbrains.com/' and 'https://cache-redirector.jetbrains.com/maven-central/org/jetbrains/kotlin/kotlin-stdlib-common/1.3.30-dev-532/', but 1.3.30-dev-532 seems not published.
I look below pages.

- https://teamcity.jetbrains.com/viewType.html?buildTypeId=Kotlin_dev_DeployMavenArtifacts_Nightly&tab=buildTypeHistoryList
- https://teamcity.jetbrains.com/viewType.html?buildTypeId=Kotlin_dev_DeployMavenArtifacts_Bootstrap&tab=buildTypeHistoryList
- https://teamcity.jetbrains.com/viewType.html?buildTypeId=Kotlin_dev_DeployMavenArtifacts_Snapshots&tab=buildTypeHistoryList

I found published task version 1.3.30-dev-588 from Kotlin_dev_DeployMavenArtifacts_Nightly, then I changed gradle.properties.
```
-kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_CompilerAllPlugins),number:1.3.30-dev-532,pinned:false/artifacts/content/maven
-kotlinVersion=1.3.30-dev-532
-testKotlinVersion=1.3.30-dev-532

+kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_dev_CompilerAllPlugins),number:1.3.30-dev-588,pinned:false/artifacts/content/maven
+kotlinVersion=1.3.30-dev-588
+testKotlinVersion=1.3.30-dev-588
```

Then the task is success.
```
$ ./gradlew dependencies:update

// many comments

BUILD SUCCESSFUL in 9m 52s
13 actionable tasks: 13 executed

```

I want change 1.3.30-dev-532 version to 1.3.30-dev-588.

How can i change the version  on master?

**Comment by vvlevchenko** [comment]
issue fixed with #2454

**Comment by Guang1234567** [comment]
@furyu-john

Hope it help you.  hahahaha

```gradle
#gradle.propoties

buildKotlinCompilerRepo=https://dl.bintray.com/kotlin/kotlin-dev/
kotlinCompilerRepo=https://dl.bintray.com/kotlin/kotlin-dev/
```
